### PR TITLE
Change @internal => @hidden

### PR DESCRIPTION
**Pull request by thechenky**
### Description
TypeDoc does not respect `@internal` tags and only respects `@hidden`, so changing all occurrences to `@hidden` to aid doc generation.

This complements #491.

### CODE REVIEW

File: src/providers/database.ts [hunk 1]
```diff
@@ -87,7 +87,7 @@ export function _instanceWithOptions(
 }
 
 export class InstanceBuilder {
-  /* @internal */
+  /* @hidden */
```
Comment (kevinajian) [inline src/providers/database.ts]: `/** @hidden */`

File: src/cloud-functions.ts [hunk 2]
```diff
@@ -149,7 +149,7 @@ export namespace Change {
   }
 
   /**
-   * @internal
+   * @hidden
```
Comment (kevinajian) [inline src/cloud-functions.ts]: should we make these single lined?

Comment (thechenky) [reply src/cloud-functions.ts]: Yep: https://github.com/firebase/firebase-functions/pull/511/commits/53c3a1f88882c9bbbf6e0ef4c769d31701ca6854

File: src/providers/database.ts (hunk 1 above)
Comment (thechenky) [reply src/providers/database.ts]: Good catch, fixed in https://github.com/firebase/firebase-functions/pull/511/commits/e69011bd9b8e445720d4457d3cecf760b02a600e.

**Pull request merged**